src/ctia/task/migrate_es_stores.clj

```diff
@@ -0,0 +1,229 @@
+(ns ctia.task.migrate-es-stores
+  (:require [clj-momo.lib.es
+             [document :as es-doc]
+             [index :as es-index]]
+            [clojure.string :as string]
+            [clojure.tools.logging :as log]
+            [ctia
+             [init :refer [init-store-service! log-properties]]
+             [properties :as p]
+             [store :refer [stores]]]))
+
+(def default-batch-size 100)
+
+(def add-groups
+  "set a document group to [\"tenzin\"] if unset"
+  (map (fn [{:keys [groups]
+            :as doc}]
+         (if-not (seq groups)
+           (assoc doc :groups ["tenzin"])
+           doc))))
+
+(def fix-end-time
+  "fix end_time to 2535"
+  (map
+   (fn [{:keys [valid_time]
+        :as doc}]
+     (if (:end_time valid_time)
+       (update-in doc
+                  [:valid_time
+                   :end_time]
+                  #(string/replace %
+                                   #"2535"
+                                   "2525"))
+       doc))))
+
+(def available-operations
+  "define all migration operations here"
+  {:__test (map #(assoc % :groups ["migration-test"]))
+   :default (map identity)
+   :add-groups add-groups
+   :fix-end-time fix-end-time})
+
+(defn compose-operations
+  "compose operations from a list of keywords"
+  [ops]
+  (let [operation-keys (map keyword
+                            (clojure.string/split ops #","))
```

Comment:
I wouldn't "parse" a string here and `ops` should be a list of keywords and parsed before (see other comment in `-main`).